src/main/webapp/js/userMapData.json

```diff
@@ -1045,6 +1080,7 @@
       "Miami International University of Art and Design",
       "Miami University",
       "Miami University of Ohio",
+      "Miami University Regionals",
```

Comment:
Appears to be a branch of Miami University, with two campuses allocated to them. Adding this should be fine though.